sktime/transformations/series/bootstrapping.py

```diff
@@ -0,0 +1,395 @@
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Bootstrapping methods for time series."""
+
+__author__ = ["ltsaprounis"]
+
+from copy import copy
+from typing import Tuple
+
+import numpy as np
+import pandas as pd
+from statsmodels.tsa.api import STL as _STL
+
+from sktime.transformations.base import BaseTransformer
+from sktime.transformations.series.boxcox import BoxCoxTransformer
+
+
+class UnivariateBootsrappingTransformer(BaseTransformer):
```

Comment:
this is a long class name :) what about just `BootstrapTransformer`? The univariate in the name is at least not needed imho.